pandamium_datapack/data/pandamium/functions/triggers/vote_shop.mcfunction

```diff
@@ -1,40 +1,59 @@
-execute if score @s vote_shop matches -5 if score @s vote_credits matches 25.. run scoreboard players remove @s vote_credits 25
+execute if score @s vote_shop matches -5 if score @s temp_3 matches ..35 if score @s vote_credits matches 20.. at @s as @r run loot give @p loot pandamium:head
+execute if score @s vote_shop matches -5 if score @s temp_3 matches ..35 if score @s vote_credits matches 20.. run scoreboard players remove @s vote_credits 20
 
-execute if score @s vote_shop matches -6 if score @s vote_credits matches 50.. run give @s minecraft:enchanted_golden_apple
-execute if score @s vote_shop matches -6 if score @s vote_credits matches 50.. run scoreboard players remove @s vote_credits 50
+execute if score @s vote_shop matches -6 if score @s temp_3 matches ..35 if score @s vote_credits matches 25.. run give @s minecraft:enchanted_book{StoredEnchantments:[{id:"minecraft:mending",lvl:1}]} 1
+execute if score @s vote_shop matches -6 if score @s temp_3 matches ..35 if score @s vote_credits matches 25.. run scoreboard players remove @s vote_credits 25
 
-execute if score @s vote_shop matches -7 if score @s vote_credits matches 20.. at @s as @r run loot give @p loot pandamium:head
-execute if score @s vote_shop matches -7 if score @s vote_credits matches 20.. run scoreboard players remove @s vote_credits 20
+execute if score @s vote_shop matches -7 if score @s temp_3 matches ..35 if score @s vote_credits matches 50.. run give @s minecraft:enchanted_golden_apple 1
+execute if score @s vote_shop matches -7 if score @s temp_3 matches ..35 if score @s vote_credits matches 50.. run scoreboard players remove @s vote_credits 50
 
-execute if score @s vote_shop matches -8 if score @s vote_credits matches 100.. run loot give @s loot pandamium:head
-execute if score @s vote_shop matches -8 if score @s vote_credits matches 100.. run scoreboard players remove @s vote_credits 100
+execute if score @s vote_shop matches -8 if score @s temp_3 matches ..35 if score @s vote_credits matches 100.. run loot give @s loot pandamium:head
+execute if score @s vote_shop matches -8 if score @s temp_3 matches ..35 if score @s vote_credits matches 100.. run scoreboard players remove @s vote_credits 100
 
-execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. run scoreboard players set @s home_cooldown 0
-execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. run scoreboard players remove @s vote_credits 1
+execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. run scoreboard players operation @s temp_2 = @s home_cooldown
+execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. if score @s temp_2 matches 1.. run scoreboard players set @s home_cooldown 0
+execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. if score @s temp_2 matches 1.. run scoreboard players remove @s vote_credits 1
```

Comment:
Why not switch line 44 and 45 here and use `home_cooldown` directly instead of `temp_2`? Yes, normally the order is first give the item/reward and then remove the credits, but I think having a different order is better than using a new temp variable (especially for readability) here.

Reply:
That doesn't work if the player only has 1 vote credit

Reply:
right, makes sense